lib/sudo_rails.rb

```diff
@@ -14,7 +14,8 @@ class << self
                   :layout,
                   :custom_logo,
                   :primary_color,
-                  :background_color
+                  :background_color,
+                  :render
```

Comment:
We should update this part too: https://github.com/markets/sudo_rails?tab=readme-ov-file#configuration

Reply:
Please check if the content added is enough